tfhe/src/core_crypto/entities/gsw_ciphertext.rs

```diff
@@ -153,7 +156,8 @@ use crate::core_crypto::commons::traits::*;
 /// Simply use the
 /// [`LWE decryption algorithm`](`crate::core_crypto::algorithms::decrypt_lwe_ciphertext`)
 /// on one of the LWE ciphertexts contained in the Lev ciphertext.
-#[derive(Clone, Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
+#[derive(Clone, Debug, PartialEq, Eq, serde::Serialize, serde::Deserialize, Versionize)]
+#[versionize(GswCiphertextVersions)]
 pub struct GswCiphertext<C: Container> {
```

Comment:
we can skip this one for now, we don't have constructors nor code associated

Reply:
done